content/_code-samples/build-a-wallet/js/.env

```
@@ -0,0 +1,3 @@
+CLIENT="wss://s.altnet.rippletest.net/"
+EXPLORER_NETWORK="testnet"
+SEED="sEd7B2Jm7RBhhh5i88SH38aZB8ot8Kq"
```

Even if it's just a throwaway Testnet seed, it's good practice not to have secret keys floating around the repo. Also, please end all files with a newline.

```suggestion
SEED="s████████████████████████████"

```